pkg/printers/internalversion/printers_test.go

```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
 	}
 }
 
+func TestPrintClusterTrustBundle(t *testing.T) {
+	tests := []struct {
+		bundle   certificates.ClusterTrustBundle
+		options  printers.GenerateOptions
+		expected []metav1.TableRow
+	}{
+		{
+			bundle: certificates.ClusterTrustBundle{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "k8s.io:foo:",
```

Comment:
Does adding `CreationTimestamp` make sense?

Reply:
This resource object does not use the createTime field